shell/tsconfig.default.json

```diff
@@ -17,16 +17,16 @@
     "rootDir": ".",
```

Comment:
It does not make so much sense to have a default inside a folder and the one in the root importing it which imports another one 😅 
We should make an issue where we review the TsConfigs we have around.